### ISSUE TITLE
require-param: Why does not report for all params?  

### ISSUE THREAD

**Issue by yeonjuan** [gajus/eslint-plugin-jsdoc#347]
I have a question. In using **require-param** rule.
```js
/**
*
*/
function quux (foo, bar) {

}
```
Why this code only report one error message ?
```
Missing JSDoc @param "foo" declaration.
```

This(below) is what i expected
```
'Missing JSDoc @param "foo" declaration.'
'Missing JSDoc @param "bar" declaration.'
```
So for fixing it, 
I Think code ```functionParameterNames.some``` in ```rules/requireParam.js``` should be  ```functionParameterNames.forEach```
But,, Is there any reason not to be?

**Comment by yeonjuan** [comment]
Additionally. I think this change needed for fixer implementation(#336)
